packages/twenty-front/src/modules/views/hooks/useViewBarFilterHotKeys.ts

```diff
@@ -0,0 +1,38 @@
+import { useDropdown } from '@/ui/layout/dropdown/hooks/useDropdown';
+import { useScopedHotkeys } from '@/ui/utilities/hotkey/hooks/useScopedHotkeys';
+import { useSetHotkeyScope } from '@/ui/utilities/hotkey/hooks/useSetHotkeyScope';
+import { VIEW_BAR_FILTER_DROPDOWN_ID } from '@/views/constants/ViewBarFilterDropdownId';
+
+import { useOpenVectorSearchFilter } from '@/views/hooks/useOpenVectorSearchFilter';
+import { useVectorSearchInputState } from '@/views/hooks/useVectorSearchInputState';
+import { useEffect } from 'react';
+import { ViewsHotkeyScope } from '../types/ViewsHotkeyScope';
+
+export const useViewBarFilterHotKeys = () => {
+  const { openDropdown } = useDropdown(VIEW_BAR_FILTER_DROPDOWN_ID);
+  const { setVectorSearchInputValueFromExistingFilter } =
+    useVectorSearchInputState(VIEW_BAR_FILTER_DROPDOWN_ID);
+  const { openVectorSearchFilter } = useOpenVectorSearchFilter(
+    VIEW_BAR_FILTER_DROPDOWN_ID,
+  );
+  const setHotkeyScope = useSetHotkeyScope();
+
+  useEffect(() => {
+    setHotkeyScope(ViewsHotkeyScope.ViewBarFilter);
```

Comment:
I'm not confortable with this at all, this will conflict with existing hotkey scope like table. Let's instead use the already existing hotkey scope `RecordIndexHotkeyScope.RecordIndex`, you can look at `RecordBoardHotkeyEffect`.

Please let's also turn this hook into an effect component just like `RecordBoardHotkeyEffect` otherwise any change in this hook might trigger a re-render of the whole ViewBar and its effect components, which can open the door to many bugs.